tests/sanity/ignore-2.13.txt

```diff
@@ -0,0 +1 @@
+tests/ee/roles/smoke/library/smoke_ipaddress.py shebang
```

Comment:
why is this ignore necessary?

Reply:
`ansible-test sanity` tries to force another shebang for non-module `.py` files, and does not recognize this to be a module. So I can either choose the wrong shebang - and fail the tests - (https://github.com/ansible-collections/community.hrobot/runs/6039348854?check_suite_focus=true; all other allowed shebangs also don't work), or I have to live with the sanity error (and ignore it).

Reply:
These are in directories known to the sanity test: https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_util/controller/sanity/code-smell/shebang.py#L84-L92